### PR TITLE
Fix Warnings

### PR DESCRIPTION
**Pull request by mkrus**
from compiler, clazy, krazy, clang-tidy

### CODE REVIEW

File: src/core/framegraphes/forwardrenderer.cpp [hunk 1]
```diff
@@ -700,7 +699,8 @@ void ForwardRenderer::reconfigureFrameGraph()
             effect->setSceneSize(targetSize);
 
             // add the layers from the effect to block them from being rendered in the main scene
-            for (auto layer : effect->layers())
+            const auto layers = effect->layers();
```
Comment (jcelerier) [inline src/core/framegraphes/forwardrenderer.cpp]: maybe a reference here, if one day `layers()` starts returning by `const&`  ?

File: src/core/fx/gaussianblureffect.cpp [hunk 2]
```diff
@@ -264,9 +264,9 @@ void GaussianBlurEffect::setBlurPassCount(int blurPasscount)
     m_blurPassCount = blurPasscount;
 
     //just do the simple thing for now and delete all children and recreate.
-    for (auto child : m_blurPassRoot->childNodes()) {
+    const auto childNodes = m_blurPassRoot->childNodes();
```
Comment (jcelerier) [inline src/core/fx/gaussianblureffect.cpp]: same

File: tools/assetpipelineeditor/animationwidget.cpp [hunk 3]
```diff
@@ -234,7 +234,7 @@ QVariant AnimationClipModel::data(const QModelIndex &index, int role) const
     case 1:
         switch (role) {
         case Qt::DisplayRole:
-            return QString("%1s").arg(m_clips.at(index.row()).duration, 0, 'f', 2);
+            return QString("%1s").arg(static_cast<double>(m_clips.at(index.row()).duration), 0, 'f', 2);
```
Comment (jcelerier) [inline tools/assetpipelineeditor/animationwidget.cpp]: maybe minor (and mostly asking for the sake of my personal knowledge) but isn't QStringLiteral recommended for these strings nowadays ?

File: tools/assetpipelineeditor/animationwidget.cpp [hunk 4]
```diff
@@ -304,7 +304,7 @@ void AnimationClipModel::updateDecorationIcon()
     static QPixmap playPixmap(":/icons/playing-circleH.png");
```
Comment (jcelerier) [inline tools/assetpipelineeditor/animationwidget.cpp]: wondering if this wouldn't benefit from being static const

File: tools/assetpipelineeditor/orbitcameracontroller.cpp [hunk 5]
```diff
@@ -330,31 +330,31 @@ QSize OrbitCameraController::windowSize() const
 
 void OrbitCameraController::setPanSpeed(float v)
 {
-    if (m_panSpeed != v) {
+    if (!qFuzzyCompare(m_panSpeed, v)) {
```
Comment (jcelerier) [inline tools/assetpipelineeditor/orbitcameracontroller.cpp]: is this behaviour wanted in setters ?

File: tools/cubemaptooctahedralmap/CopyOutputToSSBO.qml [hunk 6]
```diff
@@ -1,3 +1,31 @@
+/*
+    CopyOutputToSSBO.qml
+
+    This file is part of Kuesa.
+
+    Copyright (C) 2018 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
```
Comment (jcelerier) [inline tools/cubemaptooctahedralmap/CopyOutputToSSBO.qml]: shouldn't it be 2019 ? (and is there a script that updates them all somewhere?)

Comment (mkrus) [reply tools/cubemaptooctahedralmap/CopyOutputToSSBO.qml]: file was created in 2018 I think

File: tools/assetpipelineeditor/orbitcameracontroller.cpp (hunk 5 above)
Comment (mkrus) [reply tools/assetpipelineeditor/orbitcameracontroller.cpp]: yeah, we always compare new value to old one, avoid emitting change signal if value has not actually changed

File: tools/assetpipelineeditor/animationwidget.cpp (hunk 3 above)
Comment (mkrus) [reply tools/assetpipelineeditor/animationwidget.cpp]: true

File: tools/assetpipelineeditor/animationwidget.cpp (hunk 4 above)
Comment (mkrus) [reply tools/assetpipelineeditor/animationwidget.cpp]: done

File: src/core/fx/gaussianblureffect.cpp (hunk 2 above)
Comment (mkrus) [reply src/core/fx/gaussianblureffect.cpp]: done

File: src/core/framegraphes/forwardrenderer.cpp (hunk 1 above)
Comment (mkrus) [reply src/core/framegraphes/forwardrenderer.cpp]: done

**Pull request merged**